simplyblock_web/node_utils.py

```diff
@@ -1,66 +1,142 @@
 #!/usr/bin/env python
 # encoding: utf-8
+
 import json
 import logging
-import requests
-import boto3
 import re
+from typing import List, Tuple, TypedDict
+
+import boto3
+import requests
 
 from simplyblock_core import shell_utils
 import simplyblock_core.utils.pci as pci_utils
 
 
+# Type definitions
+class NVMENamespace(TypedDict, total=False):
```

Comment:
We could use pydantic types here for consistency with the web-implementation and flexibility.